### ISSUE TITLE
Add a title to the generate iframe

### ISSUE THREAD

**Issue by mark-making** [netlify/netlify-identity-widget#199]
To ensure an accessible web, and for the sake of passing accessibility audits, please can we set the title for the generated `iframe`? Looking through the code it would be similar to how the `id` is set. 

`iframe.id = "netlify-identity-widget";`

`iframe.title = "Netlify identity widget";`

**Comment by sw-yx** [comment]
pr welcome

**Comment by steveperrito** [comment]
Chrome's Lighthouse tab made me aware of this issue. I solved this in my Vue app with a simple check that seems to be working for me:

```javascript
// in main.js
import netlifyIdentity from 'netlify-identity-widget';

let netlifyIdentityWidgetIframe = document.getElementById('netlify-identity-widget');

if (!netlifyIdentityWidgetIframe) {
  netlifyIdentity.init();
}
```

**Comment by jlengstorf** [comment]
this was closed by #200
